lua/telescope/builtin/git.lua

```diff
@@ -77,6 +88,12 @@ git.bcommits = function(opts)
   }):find()
 end
 
+--- List branches for current directory, with log preview
+--- - Picker-specific default keymaps:
+---   - `<C-t>`: tracks currently selected branch
+---   - `<C-r>`: rebases currently selected branch
+---   - `<C-a>`: creates a new branch, with confirmation prompt before creation
+---   - `<C-d>`: deletes the currently selected branch, with confirmation prompt before deletion
```

Comment:
😍😍

Reply:
I'm proud of this one, glad you like it :heart: